_maps/RandomRuins/SpaceRuins/Fast_Food.dmm

```diff
@@ -497,9 +497,9 @@
 /area/ruin/space/has_grav/powered/macspace)
 "bT" = (
 /obj/structure/chair/stool/bar,
-/obj/item/toy/figure/hop,
+/obj/item/toy/figure/xo,
 /turf/open/floor/carpet,
-/area/ruin/space/has_grav/powered/macspace)
+/area/ruin/space/has_grav/powered/macspace) //Nsv13 - HoP to XO
```

Comment:
I have no idea how valid this is.

Reply:
Strange, as I didn't edit that part. Must've been a mistype.

Reply:
Ah, I see; I pasted two lines too low.